src/store/lookup.ts

```diff
@@ -31,8 +31,7 @@ export async function lookup(store: Store) {
 			await page.goto(link.url, {waitUntil: 'networkidle0'});
 		} catch {
 			Logger.error(`✖ [${store.name}] ${graphicsCard} skipping; timed out`);
-			await browser.close();
```

Comment:
I think we'll still need this since it's launching. in the beginning of each loop.

That being said. I do have another PR that will fix this altogether, but let's put this back for now :)

Thanks for all the work!!!